draft-janfred-radext-radiusdtls-bis.md

```diff
@@ -183,26 +183,24 @@ While the client may be able to deduce the operational state of the local server
 Within RADIUS, proxies typically only forward traffic between the NAS and RADIUS servers, and they do not generate their own response.
 As a result, when a NAS does not receive a response to a request, this could be the result of packet loss between the NAS and proxy, a problem on the proxy, loss between the RADIUS proxy and server, or a problem with the server.
 
-When UDP is used as a transport protocol, the absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
+The absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
```

Comment:
This isn't an issue with UDP specifically.  A lack of reply with any transport can lead to the client deducing that the proxy is unavailable.

Reply:
True; it's easier to jump to this possibly wrong conclusion on some transports than on others. DTLS and UDP are pretty much on par; with TLS the client could look at TCP state responsiveness and thus could make a more accurate guess on whether the remote end is still there.